frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -85,6 +88,18 @@ local footerTextGeneratorMap = {
         return footer:getDataFromStatistics(
             "TC: ", (left and left or footer.pages - footer.pageno))
     end,
+    mem_usage = function(footer)
+        local statm = io.open("/proc/self/statm", "r")
+        if statm then
+            local infos = statm:read("*all")
+            statm:close()
+            local rss = infos:match("^%S+ (%S+) ")
```

Comment:
What does rss stand for here? The only thing I can come up with is something like `return_statm_string`. Even though this variable is *quite* local maybe something a touch less cryptic would be better?